lib/net.cpp

```diff
@@ -15,6 +15,25 @@
 
 // https://wiki.openssl.org/index.php/SSL/TLS_Client
 
+static std::string socket_error(int err)
```

Comment:
You seem to have something named `socketerror` and something else named `socket_error`. Please find names that are more different.

Reply:
I changed this to `error_to_string()`, and the macro to `GetSocketErro()`.